blogposts/announcing-sourcegraph-3.6.md

```diff
@@ -0,0 +1,162 @@
+---
+title: 'Sourcegraph 3.6: Bitbucket Server plugin, search performance improvements, and quick links'
+author: Christina Forney
+publishDate: 2019-07-20T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.6
+heroImage: /sourcegraph-mark.png
+published: true
+---
+
+[Sourcegraph](https://about.sourcegraph.com/) is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
+
+<div style="padding-left: 2rem">
+
+[**🌎 Sourcegraph plugin provides native code intelligence to Bitbucket Server**](#)<br />
```

Comment:
```suggestion
[**🌎 Bitbucket Server integration brings code intelligence to pull requests and more**](#)<br />
```

While it is a Bitbucket Server plugin, from Sourcegraph's point of view, it is an integration.

Reply:
I wanted to make it clear that it was Sourcegraph providing the code intelligence. Also, we had code intelligence before via the browser extension, and I want to distinguish the difference. Any thoughts on how to improve that? What about 'Native code intelligence with Sourcegraph Bitbucket Server plugin'

Reply:
> What about 'Native code intelligence with Sourcegraph Bitbucket Server plugin

Yep, that's awesome!